pilot/pkg/model/service.go

```diff
@@ -591,6 +602,29 @@ func (ep *IstioEndpoint) CmpOpts() []cmp.Option {
 	return istioEndpointCmpOpts
 }
 
+// SortIstioEndpointByAddresses returns the IstioEndpoint address slices sorted by its addresses.
+func (ep *IstioEndpoint) SortIstioEndpointByAddresses() []string {
+	if ep == nil {
+		return nil
+	}
+
+	addrSlice := ep.Addresses
+	sort.Sort(sort.Reverse(sort.StringSlice(addrSlice)))
+	return addrSlice
+}
+
+// GetIstioEndpointKey returns the key of IstioEndpoint based on its addresses
+func (ep *IstioEndpoint) GetIstioEndpointKey() string {
```

Comment:
```suggestion
func (ep *IstioEndpoint) Key() string {
```

Get and IstioEndpoint are redundant, we already know we are getting something from an IstioEndpoint

Reply:
sure, done!